tee-worker/litentry/core/assertion-build/src/a8.rs

```diff
@@ -137,39 +81,14 @@ pub fn build(
 		},
 		Err(e) => {
 			error!("Generate unsigned credential failed {:?}", e);
-			Err(Error::RequestVCFailed(Assertion::A8(index_networks), e.into_error_detail()))
+			// In fact, it should never fail
+			let bounded_web3networks =
+				req.assertion.get_supported_web3networks().try_into().unwrap();
```

Comment:
One day it may panic 😂

Reply:
Noooo - then rust is broken.

The best thing I can do is use default value if `try_into` fails 😂 
If we fall into this branch, it means something is wrong when generating VC and we need to report it back to parachain, so we can't have any other error types than `RequestVCFailed`